Justfile

```diff
@@ -355,7 +355,18 @@ function terra_caseify()
 
     terra_sync-pipenv) # Synchronize the local pipenv for terra. You normally \
                        # don't call this directly
-      TERRA_PIPENV_IMAGE=terra_pipenv Terra_Pipenv sync ${@+"${@}"}
+      if ! command -v pipenv &> /dev/null; then
+        add_to_local=y justify terra setup --dir "${TERRA_CWD}/build/pipenv" --download
```

Comment:
Should there be a question about performing this step, or just go ahead and do it (as is currently written)?

Reply:
I kind of like the idea of it just automatically running. If we were going to have the user Bring Your Own Pipenv, it would be in the `local.env` file